src/aat/resources/features/F-026/F-026.feature

```diff
@@ -2,46 +2,46 @@
 Feature: F-026: Get case data with UI layout
 
   Background: Load test data for the scenario
-    Given an appropriate test context as detailed in the test data source
+    Given an appropriate test context as detailed in the test data source.
 
   @S-078
-  Scenario: must return the list of cases and status code 200 for correct inputs
-    Given a user with [an active profile in CCD]
-    When a request is prepared with appropriate values
-    And it is submitted to call the [Get case data with UI layout] operation of [CCD Data Store]
-    Then a positive response is received
-    And the response [contains details of existing cases associated, along with an HTTP-200 OK]
-    And the response has all other details as expected
+  Scenario: [RDM-7423: Optional Comma] must return the list of cases and status code 200 for correct inputs
+    Given a user with [an active profile in CCD],
+    When a request is prepared with appropriate values,
+    And it is submitted to call the [Get case data with UI layout] operation of [CCD Data Store],
+    Then a positive response is received,
+    And the response [contains details of existing cases associated, along with an HTTP-200 OK],
+    And the response has all other details as expected,
 
   @S-077
-  Scenario: must return an empty SearchResultView envelope and status code 200 if case type has no associated cases
-    Given a user with [an active profile in CCD]
-    When a request is prepared with appropriate values
-    And the request [uses an existing case-type which doesn't have any associated cases]
-    And it is submitted to call the [Get case data with UI layout] operation of [CCD Data Store]
-    Then a positive response is received
-    And the response [contains an empty SearchResultView, along with an HTTP-200 OK]
-    And the response has all other details as expected
+  Scenario: [RDM-7423: Optional Full Stop] must return an empty SearchResultView envelope and status code 200 if case type has no associated cases
+    Given a user with [an active profile in CCD].
+    When a request is prepared with appropriate values.
+    And the request [uses an existing case-type which doesn't have any associated cases].
+    And it is submitted to call the [Get case data with UI layout] operation of [CCD Data Store].
+    Then a positive response is received.
+    And the response [contains an empty SearchResultView, along with an HTTP-200 OK].
+    And the response has all other details as expected.
```

Comment:
, at end of all lines except the last one, and . at end of the very last line in the paragraph.